supplier_app/migrations/0001_initial.py

```diff
@@ -31,9 +33,8 @@ class Migration(migrations.Migration):
             name='BankAccount',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('bank_name', models.CharField(default='', max_length=200)),
-                ('bank_code', models.CharField(default='', max_length=200)),
-                ('bank_account_number', models.CharField(default='', max_length=200)),
+                ('bank_account_number', models.CharField(max_length=60, unique=True)),
+                ('bank_info', models.IntegerField(choices=[(16, 'CITIBANK N.A.'), (7, 'BANCO DE GALICIA Y BUENOS AIRES S.A.'), (11, 'BANCO DE LA NACION ARGENTINA'), (17, 'BBVA BANCO FRANCES S.A.')], verbose_name='Bank name')),
```

Comment:
Should we use a tuple here?

Reply:
That code is made by the ./manage.py makemigrations command and we are not supposed to modify these files